ReservePubKey/__tests__/handler.test.ts

```diff
@@ -129,6 +132,31 @@ describe("reserveSingleKey", () => {
 });
 
 describe("reservePubKeys", () => {
+  beforeEach(async () => {
+    jest.clearAllMocks();
+  });
+
+  test("GIVEN a working model WHEN getAllAssertionsRef fails THEN an internal error is returned", async () => {
+    const mockedContainer = mockContainer();
+    mockedContainer.mock.create.mockImplementation(mockCreatePendingLollipop);
+
+    getAllAssertionsRef.mockImplementationOnce(() =>
+      TE.left(Error("an error"))
+    );
+
+    const model = new LolliPOPKeysModel(mockedContainer.container);
+    const pubKey = aSha512PubKey;
+    const result = await handler.reservePubKeys(model)(pubKey);
+
+    expect(loggerMock.trackEvent).toHaveBeenCalledTimes(1);
```

Comment:
Can we add a `toHaveBeenCalledWith`?

Reply:
Sure. Done in [a14d8d7](https://github.com/pagopa/io-functions-lollipop/pull/36/commits/a14d8d78391a5f98b6d525db50d3b2db0e393a74), thanks